alternator/expressions.g

```diff
@@ -91,6 +91,16 @@ options {
         throw expressions_syntax_error(format("{} at char {}", err,
             ex->get_charPositionInLine()));
     }
+
+    // ANTLR3 tries to recover missing tokens, but it has a bug and leaks them.
```

Comment:
What do you mean "recover" missing tokens? I would use the verb "report", not "recover".